etc/profile-m-z/quaternion.profile

```diff
@@ -44,11 +46,21 @@ shell none
 tracelog
 
 disable-mnt
+# Add 'ignore private-bin' to 'quaternion.local' for filemanager or hyperlink support
 private-bin quaternion
 private-cache
 private-dev
 private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,mime.types,nsswitch.conf,pki,pulse,resolv.conf,selinux,ssl,X11,xdg
 private-tmp
 
-dbus-user none
+dbus-user.filter
+dbus-user.talk org.freedesktop.secrets
+# Add below line to 'quaternion.local' for native notifications
+# dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.kde.kwalletd5
+# Add below lines to 'quaternion.local' for KDE tray notifications
+# dbus-user.talk org.kde.StatusNotifierWatcher
+# dbus-user.talk org.kde.StatusNotifierItem-13-1
```

Comment:
- StatusNotifierItem must be a own rule, it must not?
- The pid and id in StatusNotifierItem are not stable, use wildcards.

```suggestion
# dbus-user.own org.kde.*
# dbus-user.talk org.kde.StatusNotifierWatcher
```

Reply:
Yes, it is a own rule.